dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -564,6 +569,10 @@ private String parseSecondaryFile(String stepDockerRequirement, String secondary
             List<Object> cltRequirements = null;
             List<Object> cltHints = null;
 
+            // CWLAvro only supports requirements and hints as an array, must be converted
+            entryJson = convertJSONObjectToArray("requirements", entryJson);
```

Comment:
You could create constants for "requirements" and "hints", since they're each referenced twice in the file, but not worth going through another iteration of the PR.